pkg/model/run.go

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright © 2019 One Concern
+ *
+ */
+
+package model
+
+import (
+	"time"
+)
+
+type Run struct {
+	Type        string      `json:"type" yaml:"type"` // Type of run
+	Name        string      `json:"name" yaml:"name"` // Name for the run
+	ID          string      `json:"id" yaml:"id"`     // Unique id for an instance of a run. Each time the same Type+Name is run it results in a new ID.
+	Timestamp   time.Time   `json:"timestamp" yaml:"timestamp"`
+	Contributor Contributor `json:"contributor" yaml:"contributor"`
+	Steps       []Steps     `json:"steps" yaml:"steps"`
```

Comment:
Steps here are an array, meaning they are ordered linearly (totally ordered). What about runs with complex DAGs that include steps running in parallel that cannot be ordered linearly in this way?

Reply:
Thanks, will include it.